images/spark-operator/config/main.tf

```diff
@@ -1,17 +1,32 @@
 variable "extra_packages" {
   description = "Additional packages to install."
   type        = list(string)
-  default     = ["spark-operator"]
+  default     = ["spark", "spark-operator"]
 }
 
-module "accts" { source = "../../../tflib/accts" }
+module "accts" {
+  source = "../../../tflib/accts"
+}
 
+# TODO: procps should be moved to a runtime dependency of the spark-operator
```

Comment:
Is this todo still applicable?

Reply:
Thanks for spotting - this wasn't required in the end - removed the package but not the comment